scripts/release.js

```diff
@@ -0,0 +1,35 @@
+const path = require("path");
```

Comment:
I've done my best to test this before creating the PR - but as with any CI work... i could miss something and we'll have to supervise the first automated run of this script